terraform/helm/fullnode/templates/backup-verify.yaml

```diff
@@ -46,7 +46,7 @@ spec:
             - name: STRUCT_LOG_TCP_ADDR
               value: {{ include "backup.loggingAddress" $ | quote }}
             - name: PUSH_METRICS_ENDPOINT
-              value: "{{- include "backup.pushGateway" $ }}/metrics/job/{{- .Values.backup_verify.jobName | default "db_backup_verify" }}"
+              value: "{{- include "backup.pushMetricsEndpoint" $ }}/api/v1/import/prometheus?extra_label=role={{- .jobName | default "db_backup_verify" }}&extra_label=kubernetes_pod_name={{ include "backup.fullname" . }}-db-backup-verify"
```

Comment:
This won't actually pick up the kubernetes pod name, since k8s will add a random suffix. You might need to export to a new environment variable like below, and add it to the corresponding tooling to pick up optional environment variables to add as labels.


```
       - name: KUBERNETES_POD_NAME
          valueFrom:
            fieldRef:
              fieldPath: metadata.name
```

Reply:
I don't need to distinguish between the runs though, as long as there's only one running in most if all cases?

Reply:
Then rename it something that's not `kubernetes_pod_name` to avoid confusion?